### PR TITLE
Fixes for OD4266 and OD4267

### PR DESCRIPTION
**Pull request by marxprop**
- Fix incorrect Batchnorm dimensions in torch lenet model.
- Remove duplicated model types in bokeh
- Fix tensorboard error when it's disabled and ensure model graph is created once.

### CODE REVIEW

File: dgbpy/torch_classes.py [hunk 1]
```diff
@@ -424,8 +423,9 @@ def __init__(self,
 
         self.cbs = []
         defaultCBS = [ TrainEvalCallback(), AvgStatsCallback(metrics),
-                        EarlyStoppingCallback(earlystopping), TensorBoardLogCallback()]
+                        EarlyStoppingCallback(earlystopping) ]
         if not hasFastprogress(): self.silent = True
```
Comment (waynegm) [inline dgbpy/torch_classes.py:427]: Should this be removed if we will be using a Bokeh progressbar?

Comment (marxprop) [reply dgbpy/torch_classes.py:427]: Yes, all these will slightly change when the bokeh progress bar is added.

Fastprogress is currently the default for handling the notebook and terminal progress bar during training.

**Review by arnauddgbes — APPROVED** [review]
